images/hook-reverse-proxy/Dockerfile

```
@@ -0,0 +1,4 @@
+FROM registry.access.redhat.com/ubi9/ubi-minimal:latest
+
+ADD hook-reverse-proxy /usr/bin/hook-reverse-proxy
```

Hadolint (2.14.0)

[error] Use COPY instead of ADD for files and folders

(DL3020)